runtime/js/98_global_scope_window.js

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { core, primordials } from "ext:core/mod.js";
+const ops = core.ops;
+const {
+  ObjectDefineProperties,
+  ObjectPrototypeIsPrototypeOf,
+  SymbolFor,
+} = primordials;
+
+import * as util from "ext:runtime/06_util.js";
+import * as location from "ext:deno_web/12_location.js";
+import * as console from "ext:deno_console/01_console.js";
+import * as webidl from "ext:deno_webidl/00_webidl.js";
+import * as globalInterfaces from "ext:deno_web/04_global_interfaces.js";
+import * as webStorage from "ext:deno_webstorage/01_webstorage.js";
+import * as prompt from "ext:runtime/41_prompt.js";
+
+class Navigator {
+  constructor() {
+    webidl.illegalConstructor();
+  }
+
+  [SymbolFor("Deno.privateCustomInspect")](inspect, inspectOptions) {
+    return inspect(
+      console.createFilteredInspectProxy({
+        object: this,
+        evaluate: ObjectPrototypeIsPrototypeOf(NavigatorPrototype, this),
+        keys: [
+          "hardwareConcurrency",
+          "userAgent",
+          "language",
+          "languages",
+        ],
+      }),
+      inspectOptions,
+    );
+  }
+}
+
+const navigator = webidl.createBranded(Navigator);
+
+function memoizeLazy(f) {
+  let v_ = null;
+  return () => {
+    if (v_ === null) {
+      v_ = f();
+    }
+    return v_;
+  };
+}
+
+const numCpus = memoizeLazy(() => ops.op_bootstrap_numcpus());
+const userAgent = memoizeLazy(() => ops.op_bootstrap_user_agent());
+const language = memoizeLazy(() => ops.op_bootstrap_language());
+
+ObjectDefineProperties(Navigator.prototype, {
+  gpu: {
+    configurable: true,
+    enumerable: true,
+    get() {
+      webidl.assertBranded(this, NavigatorPrototype);
+      loadWebGPU();
+      return webgpu.gpu;
```

Comment:
I added a `00_init.js` in ext/webgpu in #21773 so we can reuse `loadWebGPU()` from many places.